options.rb

```diff
@@ -0,0 +1,27 @@
+class Options
+  def display
+    puts 'Please type the number of your selection'
+    option = gets.chomp.to_i
+
+    select(option)
+  end
+
+  def select(option)
+    choice = {
+      1 => 'book_list',
+      2 => 'people_list',
+      3 => 'create_person',
+      4 => 'create_book',
+      5 => 'create_rental',
+      6 => 'rentals_list',
```

Comment:
- [ ] Please while refactoring your code your app should still work in the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) but here is the case when listing rental the app doesn't list the details of the rental correctly. As you can see from the screenshot below, the app lists the rental `book` class and the `person` class instead of listing the book title and the name of the person who created the rental as it is in the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ)
  
  ![Screenshot from 2023-03-13 20-40-56](https://user-images.githubusercontent.com/92303891/224828854-13f378ba-8d8c-420c-8b62-a1f46e1750ab.png)